Tupfile

```diff
@@ -0,0 +1,3 @@
+include_rules
+
+: cv.tex	|> !xelatex	|> %B.pdf %B.out %B.aux %B.log
```

Comment:
It won't pollute the sources, everything will be build in the `build` folder, due to the `tup.config`.